public/locales/en/translation.json

```diff
@@ -263,15 +263,15 @@
         "tooSickNumberOfDays": "If yes, enter the number of days (1-7) you were unable to work.",
         "help-tooSickNumberOfDays": "You must report the number of days that work was missed.",
         "couldNotAcceptWork": "Was there any reason (other than sickness, injury or the disaster) that you could not have accepted full-time work, each workday?",
-        "help-couldNotAcceptWork": "Other than being sick, injured or disaster, could full-time work have been accepted?",
-        "didYouLook": "Did you look for work or contact your last employer, or, if self employed, did you attempt to resume self- employment?",
-        "help-didYouLook": "The work search requirements on your notice of Unemployment Insurance Award must be followed. Work Searches may include in-person, mail, telephone, or Internet contacts with employers. Was there any attempt to look for work or resume work?",
+        "help-couldNotAcceptWork": "Other than being sick, injured, or the disaster, could full-time work have been accepted? ",
```

Comment:
is the extra space at the end intentional? just checking

Reply:
No! Thank you!